legion/sdk/legion/sdk/clients/templates.py

```diff
@@ -0,0 +1,23 @@
+import os
```

Comment:
LICENSE header? Function's documentation is required by pydoc

Reply:
Fixed